releases/handbook.md

```diff
@@ -1,193 +1,71 @@
+* Week 4 - (Software development Phase) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting)
+* Week 6 - (Software development Phase) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting)
+* Week 8 - (Last week of Software development Phase) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting)
+* Week 9 - (Feature Freeze and Prep for Release) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting). Discuss items required to prepare for Release (including documentation).
+* Week 10 - (Community and Distribution testing starts) Users and distributions run their automated test suites and try to stay away from manual labor-intensive tests.
+* Week 11 - (Bug Fixing) - Discuss any potential blockers for the Release.
+* Week 12 - (Release) - Items to discuss: bug fixes required, release cut, documentation needed.
+- Manifests WG leads synchronizes and cut the release on Kubeflow Platform/Manifests.
+- The Release Manager is responsible for approving/reviewing the blog and slides announcing the release.
 
 ## People and Roles
 
-### Release Manager
-The Release Manager will be responsible for coordinating the release and taking ultimate accountability for all release tasks to be completed on time.
-
-The previous Release Manager is responsible for choosing their successor that meets the prerequisite. If multiple Release Team members meet the prerequisites to become a Release Manager, the one from a different organization than the one that drove the previous release should be preferred.
-
-**Prerequisites:**
-1. Have contributed as a release team member in two or more previous releases
-
-**Time Commitments:** This role will need to be synchronous. The Release Manager is expected to join most of the weekly meetings and coordinate with the Release Team and the Working Group leads. For the Development phase, an estimate of the work needed is 4-8 hours per week. For the next phases, the Release Manager is required to devote more time and coordinate the tasks of the release.
-
-**Responsibilities:**
-* Provide updates to the mailing list with the progress of the release
+### Release Management Team: 
+The Release Management Team is composed by: Release manager, WG leads / liasons, technical documentation lead and product manager.
+Release team liaisons will be responsible not for the communication but for contributing to the release with documentation, source code, PRs review, etc, according to their skills and motivation.
+
```

Comment:
- The previous Release Manager is responsible for choosing their successor that meets the prerequisite.